api/attendance/getAllAttendanceChanges.ts

```diff
@@ -0,0 +1,27 @@
+
+
+
+import { VercelRequest, VercelResponse } from '@vercel/node'
+import { isEmpty } from "../../src/utils"
+import { z, ZodError } from "zod";
+import { AttendanceQuery } from "../../src/types/types"
+import AttendanceController from "../../src/controllers/attendanceController"
+
+const attendanceController = new AttendanceController();
+
+export default async (req: VercelRequest, res: VercelResponse) => {
+  try {
+    let people;
+    if (isEmpty(req.query)) {
+      people = await attendanceController.getAllAttendanceChanges();
```

Comment:
more of a nitpick, Also I realize this is my old naming convention (sorry), but could you change this variable name from people to attendance since people doesn't really make much sense here